front/app/containers/Admin/projects/project/analysis/InputPreview/index.tsx

```diff
@@ -40,7 +43,11 @@ const InputListItem = () => {
       {analysis.data.attributes.participation_method === 'ideation' && (
         <Box display="flex" justifyContent="flex-end">
           <Button
-            linkTo={`/admin/projects/${analysis.data.relationships.project?.data?.id}/ideas?selected_idea_id=${selectedInputId}`}
+            linkTo={`/admin/projects/${
+              analysis.data.relationships.project?.data?.id
+            }${
+              phaseId ? `/phases/${phaseId}` : ''
+            }/ideas?selected_idea_id=${selectedInputId}`}
```

Comment:
This check should move higher up the component tree, right? The button should not render if there's no `phaseId`. The link won't work in that case. (Although each input should have a phase.)

Reply:
Good point!